lib/assets/i18n/log/messages_en.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Ignore the following for cluster support testing 

(This is testing for the logger messages substitution due to my unclean branch)

Reply:
Could we not zXXX here?
A discussion needs to occur on what Apps can have as IDs.
It may be that zXXX is OK, but we don't know yet. Please skip message IDs for apps for now.